account_fiscal_position_partner_type/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- encoding: utf-8 -*-
+# Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import res_company
+from . import res_partner
+from . import account_fiscal_position
```

Comment:
I think this one should be before the others, as it's used for getting the selection. Anyways, it can be possible that the used trick doesn't serve for this version.

Reply:
I think its Ok. I needed to add a hook to pass an error when installing the module.